cli/bench_command.go

```diff
@@ -63,31 +66,32 @@ const (
 func configureBenchCommand(app commandHost) {
 	c := &benchCmd{}
 	bench := app.Command("bench", "Benchmark utility").Action(c.bench)
-	bench.Arg("subject", "Subject to use for testing").Required().StringVar(&c.subject)
+	bench.Arg("subject", "Subject to use for the benchmark").Required().StringVar(&c.subject)
 	bench.Flag("pub", "Number of concurrent publishers").Default("0").IntVar(&c.numPubs)
 	bench.Flag("sub", "Number of concurrent subscribers").Default("0").IntVar(&c.numSubs)
+	bench.Flag("js", "Use JetStream streaming").Default("false").BoolVar(&c.js)
+	bench.Flag("request", "Request/Reply mode: publishers send requests waits for a reply").Default("false").BoolVar(&c.request)
+	bench.Flag("reply", "Request/Reply mode: subscribers send replies").Default("false").BoolVar(&c.reply)
```

Comment:
I also prefer Request/Repmy but I think nats docs etc mostly say Request-Reply

Reply:
with a / is how I have always written it (for many decades now :)), but you are right that the docs have a mixture of - and /. As this made me curious and I did some research about it and as it turns out the hyphenation is actually correct here because because it represents the interaction of a request _and_ a reply while the slash is the one to use to convey that it can be a request _or_ a reply.
So you learn something new every day and I will go with the hyphen (and also harmonize the docs while I'm at it)